gym/lib/gym/runner.rb

```diff
@@ -162,6 +167,23 @@ def copy_mac_app
       app_path
     end
 
+    # Make the .pkg from .app
+    def make_mac_pkg(app_path)
+      sign = Gym.config[:installer_codesigning_identity].shellescape
+      UI.crash!("Couldn't find Developer Installer Code Signing Identity") unless sign
+
+      app_name = Gym.project.app_name.shellescape
+      pkg_name = app_name << ".pkg"
+      pkg_path = File.join(Gym.config[:output_directory], pkg_name)
+
+      command = "productbuild --sign #{sign} --component #{app_path} /Applications #{pkg_path}"
```

Comment:
MMm I now see you shellescape some parameters but not all (pkg_path e.g.). Also app_path is shellescaped before the call to `make_mac_pkg` and has some side effects.